File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ListingFullResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ListingFullResponse 
{
    public long Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public long Start { get; set; }
    public long End { get; set; }
    public string LocationName { get; set; }
    public string LocationDescription { get; set; }
    public bool PrivateListing { get; set; }
    public bool Active { get; set; }
    public List<ParticipantResponse> Participants { get; set; }
    public AccountShortResponse Owner { get; set; }
    public List<FilterResponse> Filters { get; set; }
}

File: server-scala-play-server/app/model/ListingFullResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ListingFullResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ListingFullResponse(
  id: Option[Long],
  name: Option[String],
  description: Option[String],
  start: Option[Long],
  end: Option[Long],
  locationName: Option[String],
  locationDescription: Option[String],
  privateListing: Option[Boolean],
  active: Option[Boolean],
  participants: Option[List[ParticipantResponse]],
  owner: Option[AccountShortResponse],
  filters: Option[List[FilterResponse]]
)

object ListingFullResponse {
  implicit lazy val listingFullResponseJsonFormat: Format[ListingFullResponse] = Json.format[ListingFullResponse]
}

